tests/test_dates.py

```diff
@@ -37,3 +39,10 @@ def test_get_formatted_date_returns_malformed_dates(self):
         # Strings not in the correct format or invalid dates should be returned as given and a exception logged
         self.assertEqual(get_formatted_date(''), '')
         self.assertEqual(get_formatted_date('1999-12-32 23:59:59'), '1999-12-32 23:59:59')
+
+    def test_get_current_date_bst(self):
+        # Test must be updated for last Sunday in March/October to reflect BST
```

Comment:
Would it be better to work out whether we're in GMT or BST based on the current date (i.e. BST if the current date is between the last Sunday in March and the last Sun in Oct, otherwise GMT)?  It would save the tests having to be changed every 6 months.

Reply:
changes coming, sorry should hve put a label on this